app/views/assignments/show.html.erb

```diff
@@ -89,3 +92,13 @@
     <% end %>
   </div>
 </div>
+
+<script>
```

Comment:
This script runs every time the partial loads. On a hard refresh, `roster_tab.js` hasn't loaded yet so we need to check to see if the function exists. On a turbolinks load, the script does exist.

Reply:
Let's extract it to it's own partial cause we need it for the group assignments show page as well.

Reply:
Right, group assignments exist -_-